products-service/src/functions/seedProductsService/handler.ts

```diff
@@ -0,0 +1,57 @@
+import 'source-map-support/register';
+
+import { Pool } from 'pg';
+import dbConfig from '../../dbConfig';
+
+let pool;
+
+const seedProductService = async () => {
+  if (!pool) {
+    pool = new Pool(dbConfig);
+  }
+
+  const client = await pool.connect();
+
+  try {
+    await client.query(`
+      CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+      CREATE TABLE IF NOT EXISTS products
+      (
+        id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+        title VARCHAR NOT NULL,
+        description VARCHAR,
+        price INTEGER
+      );
+      
+      CREATE TABLE IF NOT EXISTS stocks
+      (
+        product_id UUID REFERENCES products (id),
```

Comment:
you'd better add some 'id' as a primary key

Reply:
Had the same thoughts. But decided to leave structure according to the task description.